pyproject.toml

```diff
@@ -31,3 +31,6 @@ addopts = """\
     --ds=tests.settings
     """
 django_find_project = false
+
+[tool.check-manifest]
+ignore = ["docs/Makefile", "docs/make.bat", "docs/source/*.py", "docs/source/*.rst", ".readthedocs.yaml", "docs/source/spelling_wordlist.txt"]
```

Comment:
this is not the right solution for check-manifest failures - the right solution is to update `MANIFEST.in`